src/card.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
Move it too. It doesn't apply to the card block. You can put it to style.css or normalize.css